src/swap/components/SwapSettings.tsx

```diff
@@ -57,7 +57,7 @@ export function SwapSettings({
           )}
           onClick={handleToggle}
         >
-          {iconSvg}
+          <div className="h-[1.125rem] w-[1.125rem]">{iconSvg}</div>
```

Comment:
Moving SVG sizing to be handled by the parent component. 

This fixes a bug where a custom icon would be unrestrained

Reply:
Will add these src file changes in a separate PR and merge first

Reply:
Merge first https://github.com/coinbase/onchainkit/pull/1273